src/app.js

```diff
@@ -11,5 +12,6 @@ dbase.authenticate().then(() => {
 });
 
 const app = express();
+app.use('/api/v2', routes);
```

Comment:
add user routes here as well. into routes so that only one route object is imported here

Reply:
Done